### ISSUE TITLE
Pods Naming and Delete issues

### ISSUE THREAD

**Issue by SJKCoder** [pods-framework/pods#4257]
If you create a Custom Post Type with the name     restricted-documents     and one with the name  restricted-documents-type   then, instead of creating the second one, it renames the first. I tried to resolve this by changing the   labels of the first Pod back to restricted-document. That seemed to work.    I then tried to create again the second one  but with another name  restricted-documenttype. That seemed to work. But then: As I tried to delete the first, it says, Pod not found. You can not delete the Pod

**Comment by pglewis** [comment]
https://developer.wordpress.org/reference/functions/register_post_type/

```
$post_type
(string) (Required) Post type key. Must not exceed 20 characters and may only contain lowercase alphanumeric characters, dashes, and underscores.
```

**Comment by SJKCoder** [comment]
Then at least give a warning if the used key naming is invalid

**Comment by pglewis** [comment]
Absolutely, it's unacceptable if we're not already doing that.  We need to verify the behavior and fix that if it's not the case.  However priority 1 is always making sure you can keep working.

**Comment by sc0ttkclark** [comment]
We have some limiting but it should be dynamic based on the pod type selected via JS:
https://github.com/pods-framework/pods/blob/2.x/ui/admin/setup-add.php#L179

In the future, we should be able to pass back an error too.
